src/layout/FileUploadWithTag/FileListComponent.tsx

```diff
@@ -278,15 +258,17 @@ export function FileList(props: FileListProps): JSX.Element | null {
                       align='right'
                       key={`edit-${index}`}
                     >
-                      <IconButton
-                        style={{ color: 'black' }}
+                      <Button
+                        size={ButtonSize.Small}
+                        variant={ButtonVariant.Quiet}
+                        color={ButtonColor.Secondary}
                         onClick={() => props.onEdit(index)}
-                        tabIndex={0}
-                        className={classes.editTextContainer}
+                        icon={<PencilIcon title={getLanguageFromKey('general.edit_alt', props.language)} />}
+                        iconPlacement='right'
+                        className={css.customStyleEditButton}
                       >
                         {getLanguageFromKey('general.edit_alt', props.language)}
-                        <i className={`fa fa-editing-file ${classes.editIcon}`} />
-                      </IconButton>
+                      </Button>
```

Comment:
Have you tested this with a screen reader to make sure it does not read the text twice? When the button has text you can also just set aria-hidden on the icon instead of title.